setup.py

```diff
@@ -6,19 +6,25 @@
 with open('README.rst') as readme:
     description = readme.read()
 
+
+with open('HISTORY.rst') as history:
+    changelog = history.read()
+
+
 setup(
     name='dj-saml-idp',
     version=saml2idp.__version__,
     author='Sebastian Vetter',
     author_email='sebastian@mobify.com',
     description='SAML 2.0 IdP for Django',
-    long_description=description,
+    long_description='\n\n'.join([description, changelog]),
     install_requires=[
         'Django>=1.4',
         # We have to pin M2Crypto to version 0.22.3 because more recent
         # versions are failing due to issues with finding openssl libs.
         'M2Crypto==0.22.3',
-        'BeautifulSoup>=3.2.0'],
+        'BeautifulSoup>=3.2.0',
+        'structlog'],
```

Comment:
I generally prefer not making any suggestions about dependencies at all. The only reason we are doing it for M2Crypto and BeautifulSoup is that there are known issues with specific  version.